sympy/matrices/tests/test_determinant.py

```diff
@@ -12,14 +12,27 @@ class DeterminantOnlyMatrix(_MinimalMatrix, _CastableMatrix, MatrixDeterminant):
     pass
 
 def test_determinant():
+    M = Matrix()
 
-    for M in [Matrix(), Matrix([[1]])]:
```

Comment:
I superseded the `[Matrix(), Matrix([[1]])]` test with a couple of more explicit tests - we want to check that the determinant of the `[[x]]` matrix is `x` (as opposed to 'always `1`'), and the determinant of the `[]` matrix is always `1`.